lib/spaceship/tunes/app_version.rb

```diff
@@ -273,6 +334,37 @@ def supports_apple_watch
         (super != nil)
       end
 
+      # GeoJson optional values
+      def app_geojson_original_name
+        v=raw_data['transitAppFile']['value']
+        v['name'] if v
+      end
+
+      def app_geojson_original_name=(value)
+        v=raw_data['transitAppFile']['value']
+        v['name']=value if v
+      end
+
+      def app_geojson_token
+        v=raw_data['transitAppFile']['value']
+        v['assetToken'] if v
+      end
+
+      def app_geojson_token=(value)
+        v=raw_data['transitAppFile']['value']
```

Comment:
If not, you could use `raw_data.fetch('transitAppFile', {}).fetch('value', nil)`